### ISSUE TITLE
color ring for flood

### ISSUE THREAD

**Issue by lindsaycarr** [USGS-VIZLAB/gage-conditions-gif#98]
test with just 95% only to see if it visually works

**Comment by lindsaycarr** [comment]
Testing green ring around sites that are "flooding" (I used >= 97%). I don't think green is the answer, but I also don't know if showing this extra layer of information is useful. Still thinking about it.

![ezgif com-video-to-gif 1](https://user-images.githubusercontent.com/13220910/51871217-652c4500-231b-11e9-8853-47d8376a9b4f.gif)

**Comment by lindsaycarr** [comment]
I also just tested out growing the size of a dot+highlight with a ring around it if it's "flooding" (using >0.98% as the "flood" threshold for an example). Chad suggested emphasizing regions with actual flooding and regions with peak low flows (I'm thinking maybe filled in or circle with x through the center, pch = ) to highlight the very extreme points. I have been thinking about this more and more, and now after having tested out the flooding example, I think it might be trying to add too much to the map. I think maybe we should stick with just saying wet/dry for this one and look at adding those two features in our interactive version.

TLDR: indicating gages that are actually flooding + peak low flows can be used in the interactive version instead of this video version.

**Comment by lindsaycarr** [comment]
![image](https://user-images.githubusercontent.com/13220910/52085257-a154e500-2569-11e9-99d8-f3d6e463cfd7.png)

Obviously would need to fiddle with the color/display. See this branch for the code: https://github.com/lindsaycarr/gage-conditions-gif/tree/show-floods

**Comment by lindsaycarr** [comment]
And now that I am thinking about it today, the goal is to summarize WY2018 and I think we would be doing a disservice to it if we didn't indicate flooding vs just higher levels (and peak or not..).

@aappling-usgs do you have any thoughts? If we go with showing flooding, I still need to choose a better way to show it.

**Comment by aappling-usgs** [comment]
Based on your test, I think showing flood status is visually achievable. I agree with your recent thinking that this would be a really nice feature to include. My only hesitation might be that I think not all gages have flood stage information, is that right? If a big fraction (>10%, say?) of the dots being displayed have no flood stage, then I'd be more reluctant to include that info for the other ones, though we could get around that by noting somewhere that not all gages have that info available.

I often drift to purples when I'm using red and blue and need a third color, and that's what I'd try next here. But whether you land on purple, green, or something else, I think the ring color should be a little brighter so it stands out more from a distance.

How about keeping the dot the same size as the others but adding a thick ring around it (thicker than what you currently have)? Idea would be that yes, the inside of the dot might be dark or lighter blue to stay true to the data, but the main message would be the presence of the ring=FLOOD!.

There could still be just one legend entry for flooding, just showing the darkest blue and its ring, even if the inside dot might occasionally be light on the map itself, again going with the idea that we have a main message about flooding and the details of edge cases are less important.

**Comment by aappling-usgs** [comment]
I'm inclined to forge ahead with a note.
